Edits to sklearn.rst

oleksandr-pavlyk opened the pull request:
1. Made table use class wide, to span two columns
2. Made English grammar, fluency edits
3. Cleaned the code to fit the width of the column,
   shortened the amount of vertical space it requires

oleksandr-pavlyk committed:
Edits to sklearn.rst

1. Made table use class wide, to span two columns
2. Made English grammar, fluency edits
3. Cleaned the code to fit the width of the column,
   shortened the amount of vertical space it requires

anton-malakhov commented on papers/oleksandr_pavlyk/sklearn/sklearn.rst:
```diff
@@ -28,33 +27,33 @@ We prepared several benchmarks to demonstrate performance that can be achieved w
+              help="iteration", type=int, default=10)
+        argParser.add_argument('-p', '--proc', default=-1,
+              help="n_jobs for algorithm", type=int)
         args = argParser.parse_args()
```
I'd actually say that in the paper the full source is an overkill, it should rather refer to github repo and show only important parts which are necessary for understanding. Approving now since it's of course better than what we have now

oleksandr-pavlyk replied:
I agree

anton-malakhov commented on papers/oleksandr_pavlyk/sklearn/sklearn.rst:
```diff
@@ -1,19 +1,18 @@
+Specifically, update 2 optimizes Principal Component Analysis (PCA), Linear and Ridge Regressions, Correlation and Cosine Distances, and K-Means. Speedups may range from 1.5x to 160x.
 
-There are no exact matching between sklearn and DAAL API and they aren’t fully compatible for all inputs so for the cases when daal4sklearn detects incompatibility it fallbacks to original sklearn’s implementation.
+There are no direct matching between sklearn's and DAAL's APIs and they aren’t fully compatible for all inputs, therefore in cases when daal4sklearn detects incompatibility it fallbacks to original sklearn’s implementation.
```
it might need full name like `API of Intel |R| DAAL`  for compliance

anton-malakhov commented on papers/oleksandr_pavlyk/sklearn/sklearn.rst:
```diff
@@ -90,10 +84,11 @@ We prepared several benchmarks to demonstrate performance that can be achieved w
 
-Using all 32 cores of Xeon E5-2698v3 IDP’s KMeans can be faster more than 50 times comparing with python available on Ubuntu 14.04.
-P below means amount of CPU cores used.
+Using all 32 cores of Xeon E5-2698 v3 IDP’s K-Means can be faster more than 50 times comparing with python available on Ubuntu 14.04.
```
This is another TM issue here

oleksandr-pavlyk committed:
Further edits to sklearn.rst

1. Addressed reviewer's comments about use of |R| and |TM|.
2. Further streamlining of English
